controllers/azuremanagedcontrolplane_reconciler.go

```diff
@@ -45,10 +45,6 @@ type azureManagedControlPlaneService struct {
 
 // newAzureManagedControlPlaneReconciler populates all the services based on input scope.
 func newAzureManagedControlPlaneReconciler(scope *scope.ManagedControlPlaneScope) (*azureManagedControlPlaneService, error) {
-	managedClustersSvc, err := managedclusters.New(scope)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
I'd kinda like the linter to send you some flare "nice job reducing a little bit of conditional complexity"